docs/resources/cce_node_attach.md

```diff
@@ -0,0 +1,150 @@
+  flavor_id                   = "sn3.large.2"
+  availability_zone           = data.huaweicloud_availability_zones.test.names[0]
+  key_pair                    = huaweicloud_compute_keypair.test.name
+  delete_disks_on_termination = true
+  
+  system_disk_type = "SAS"
+  system_disk_size = 40
+  
+  data_disks {
+	  type = "SAS"
+	  size = "100"
+  }
+  
+  network {
+	  uuid = huaweicloud_vpc_subnet.test.id
+  }
+}
+
+resource "huaweicloud_cce_node_attach" "test" {
+  cluster_id = huaweicloud_cce_cluster.test.id
+  server_id  = huaweicloud_compute_instance.test.id
+  key_pair   = huaweicloud_compute_keypair.test.name
+  os         = "EulerOS 2.5"
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+
+```
+
+## Argument Reference
+The following arguments are supported:
+
+* `cluster_id` - (Required, String, ForceNew) Specifies the ID of the cluster.
+  Changing this parameter will create a new resource.
+
+* `name` - (Optional, String) Specifies the Node Name.
+
+* `server_id` - (Required, String, ForceNew) Specifies the flavor id.
+  Changing this parameter will create a new resource.
+
+* `os` - (Required, String, ForceNew) Specifies the operating System of the node.
+  Changing this parameter will create a new resource.
+    - For VM nodes, clusters of v1.13 and later support *EulerOS 2.5* and *CentOS 7.6*.
+
+* `key_pair` - (Optional, String, ForceNew) Specifies the key pair name when logging in to select the key pair mode.
+    This parameter and `password` are alternative. Changing this parameter will create a new resource.
+
+* `password` - (Optional, String, ForceNew) Specifies the root password when logging in to select the password mode.
+    This parameter must be salted and alternative to `key_pair`. Changing this parameter will create a new resource.
+
+* `max_pods` - (Optional, Int, ForceNew) Specifies the the maximum number of instances a node is allowed to create.
+    Changing this parameter will create a new resource.
+
+* `docker_base_size` - (Optional, Int, ForceNew) Specifies the available disk space of a single docker container
+  on the node in device mapper mode. Changing this parameter will create a new resource.
+
+* `lvm_config` - (Optional, String, ForceNew) Specifies the docker data disk configurations.
+  The following is an example:
+```hcl
+  lvm_config = "dockerThinpool=vgpaas/90%VG;kubernetesLV=vgpaas/10%VG"
+```
+  Changing this parameter will create a new resource.
+
+* `preinstall` - (Optional, String, ForceNew) Specifies the script required before installation.
+  The input value can be a Base64 encoded string or not.
+  Changing this parameter will create a new resource.
+
+* `postinstall` - (Optional, String, ForceNew) Specifies the script required after installation.
+  The input value can be a Base64 encoded string or not.
+  Changing this parameter will create a new resource.
+
+* `image_id` - (Optional, String, ForceNew) Specifies the image ID. Changing this parameter will create a new resource.
```

Comment:
what's the difference between `os` and `image_id`?

Reply:
This parameter specifies a custom image ID while adding a BMS node. It won't work as Adding a BMS node is not supported for now. Remove is from the doc could be better